Naming convention check

Raxnam opened the pull request:
updated files to match naming convention and added a guideline file for future scripts to follow

Commit:
refactor(file naming) Updated files to match naming convention

updated file names and class names to fit the naming convention

Commit:
docs(naming convention) Added naming convention ReadMe

added a file to outline the naming conventions

andrew-hirata-playeveryware commented on Assets/Scripts/EOSPeer2PeerManager.cs:
```diff
@@ -92,7 +92,7 @@ public class EOSPeer2PeerManager : IEOSSubManager
         private bool ChatDataCacheDirty;
 
-        public UIPeer2PeerParticleManager ParticleManager;
+        public UIPeer2PeerParticleController ParticleManager;
```
Should the variable itself be called something like 'ParticleManager'?

Commit:
refactor(naming changes) Renamed ParticleManager

renamed the ParticleManager in the P2P demo to ParticleController to fit better with the new names
